runners/console/index.ts

```diff
@@ -252,17 +252,20 @@ export class Console extends Runner {
         result.returnCode = 0;
         
         let filepath = path.join(this.getVariable(this.workspaceDirectory), runCommand.command.parameters[0]);
-
-        let process = ConsoleUtils.executeCommandAsync("docker-compose up", filepath, result, this.env);
-        process.on('close', (code) => {
-            if (code !== 0) {
-                result.returnCode = code;
+        if(runCommand.command.parameters.length == 2 && runCommand.command.parameters[1].port){
+            let process = ConsoleUtils.executeCommandAsync("docker-compose up", filepath, result, this.env);
+            process.on('close', (code) => {
+                if (code !== 0) {
+                    result.returnCode = code;
+                }
+            });
+            if(process.pid) {
+                this.asyncProcesses.push({ pid: process.pid, name: "dockerCompose", port: runCommand.command.parameters[1].port });
             }
-          });
-        if(process.pid && runCommand.command.parameters.length == 2) {
```

Comment:
I think we should check additionally if the port is given. Only checking if we got two parameters is not enough, because what if we give a second parameter object which does not include port. Then port will be null.

Reply:
It's already inside  a if condition, where we check if the port was given

Reply:
this is the code that was overwritten